Source/ToolCore/JSBind/JSBHeaderVisitor.h

```diff
@@ -487,10 +493,6 @@ class JSBHeaderVisitor : public SymbolVisitor
                 if (function->isPureVirtual())
                     jclass->SetAbstract();
 
-                // only want public functions
-                if (!symbol->isPublic())
-                    continue;
-
```

Comment:
We should not be processing private methods at all as they will never be bound.  This adds overhead and will process many functions which will never be used.